docs/source/quickstart.mdx

```diff
@@ -1,296 +1,509 @@
 ---
-title: Get Started
-subtitle: Run Apollo MCP Server for the first time
+title: Apollo MCP Server Quickstart
+subtitle: Create and run an MCP server in minutes with Apollo
 ---
 
-<PreviewFeature>
+Apollo MCP Server is a [Model Context Protocol](https://modelcontextprotocol.io/) server that exposes your GraphQL API operations as MCP tools.
 
-This feature is in [preview](/graphos/resources/feature-launch-stages#preview). Your questions and feedback are highly valued—don't hesitate to get in touch with your Apollo contact or post in the [Apollo Community MCP Server Category](https://community.apollographql.com/c/mcp-server/41).
+This guide walks you through the process of creating, running and configuring an MCP server with Apollo.
 
-</PreviewFeature>
+- [Step 1: Create Your MCP Server](#Step-1-Create-an-MCP-Server)
+- [Step 2: Run Your MCP Server](#Step-2-Run-Your-MCP-Server)
+- [Step 3: Connect to Claude Desktop (or another MCP client)](#Step-3-Connect-to-Claude-Desktop)
+- [Step 4: Working with MCP Tools](#Step-4-Working-with-MCP-Tools)
+- [Step 5: Deploy Your MCP Server](#Step-5-Deploy-Your-MCP-Server)
```

Comment:
These links aren't taking me to the right section. Are the anchor IDs supposed to be in lowercase?